### DISCUSSION TITLE
Regarding Downgrading MariaDB

### DISCUSSION THREAD

**Discussion by riobijoy** [snipe/snipe-it#14196] [General]
### Is your feature request related to a problem? Please describe.

Right now i am using a Snipeit version 6.2.3 and MariaDB version 11.2,But For some reason i am installing another Application in IIS and the other Application MariaDB version needed Below 11,So just i want to know Can i Downgrade my MariaDB to below 11,Is there any issue will happen in existing DB,Kindly Guide

### Describe the solution you'd like

Right now i am using a Snipeit version 6.2.3 and MariaDB version 11.2,But For some reason i am installing another Application in IIS and the other Application MariaDB version needed Below 11,So just i want to know Can i Downgrade my MariaDB to below 11,Is there any issue will happen in existing DB,Kindly Guide

**Comment by uberbrady** [comment]
We've definitely seen issues where moving from a newer DB to an older DB can run you into problems - namely, when the newer DB creates a SQL dump-file that has features that your older DB doesn't support. You can *usually* fix that by hand-editing the file to remove the offending lines. I think those are both new enough that you _probably_ shouldn't run into problems, but I would definitely do a test-restore first to make sure.
